test/gasBenchmarks/deposits.js

```diff
@@ -2,232 +2,232 @@
-        });
-    });
-
-    it('Should make Generic deposit - Centrifuge asset', async () => {
-        const depositTx = await BridgeInstance.deposit(
-            chainID,
-            centrifugeAssetResourceID,
-            Helpers.createGenericDepositData('0xc0ff33'),
-            { from: depositerAddress }
-        );
-
-        gasBenchmarks.push({
-            type: 'Generic - Centrifuge Asset',
-            gasUsed: depositTx.receipt.gasUsed
-        });
-    });
-
-    it('Should make Generic deposit - No Argument', async () => {
-        const depositTx = await BridgeInstance.deposit(
-            chainID,
-            noArgumentResourceID,
-            Helpers.createGenericDepositData(null),
-            { from: depositerAddress }
-        );
-
-        gasBenchmarks.push({
-            type: 'Generic - No Argument',
-            gasUsed: depositTx.receipt.gasUsed
-        });
-    });
-
-    it('Should make Generic deposit - One Argument', async () => {
-        const depositTx = await BridgeInstance.deposit(
-            chainID,
-            oneArgumentResourceID,
-            Helpers.createGenericDepositData(Helpers.toHex(42, 32)),
-            { from: depositerAddress }
-        );
-
-        gasBenchmarks.push({
-            type: 'Generic - One Argument',
-            gasUsed: depositTx.receipt.gasUsed
-        });
-    });
-
-    it('Should make Generic deposit - Two Arguments', async () => {
-        const argumentOne = [NoArgumentInstance.address, OneArgumentInstance.address, TwoArgumentsInstance.address];
-        const argumentTwo = Helpers.getFunctionSignature(CentrifugeAssetInstance, 'store');
-        const encodedMetaData = Helpers.abiEncode(['address[]','bytes4'], [argumentOne, argumentTwo]);
-        const depositTx = await BridgeInstance.deposit(
-            chainID,
-            twoArgumentsResourceID,
-            Helpers.createGenericDepositData(encodedMetaData),
-            { from: depositerAddress }
-        );
-
-        gasBenchmarks.push({
-            type: 'Generic - Two Arguments',
-            gasUsed: depositTx.receipt.gasUsed
-        });
-    });
-
-    it('Should make Generic deposit - Three Arguments', async () => {
-        const argumentOne = 'soylentGreenIsPeople';
-        const argumentTwo = -42;
-        const argumentThree = true;
-        const encodedMetaData = Helpers.abiEncode(['string','int8','bool'], [argumentOne, argumentTwo, argumentThree]);
-        const depositTx = await BridgeInstance.deposit(
-            chainID,
-            threeArgumentsResourceID,
-            Helpers.createGenericDepositData(encodedMetaData),
-            { from: depositerAddress }
-        );
-
-        gasBenchmarks.push({
-            type: 'Generic - Three Arguments',
-            gasUsed: depositTx.receipt.gasUsed
-        });
-    });
-
-    it('Should print out benchmarks', () => console.table(gasBenchmarks));
-});
+ const BridgeContract = artifacts.require("Bridge");
```

Comment:
@karthikeyan-cere 
You should avoid such changes like add 1 space in each line in the file. Better not even commit it locally but revert and add only relevant changes.

Reply:
Reverted the changes